script.js

```diff
@@ -303,9 +305,39 @@ function validation() {
     document.getElementById('btn-form').type = 'submit';
     text.innerHTML = '';
   } else {
-    form.classList.remove('valid');
+    // form.classList.remove('valid');
     form.classList.add('invalid');
     document.getElementById('btn-form').type = 'button';
     text.innerHTML = '* Your email is invalid. (make sure your email is in lower cases)';
   }
-}
\ No newline at end of file
+}
+
+// Local Storage
+const form = document.getElementById('form');
+const nameField = document.getElementById('name');
+const emailField = document.getElementById('email');
+const messageField = document.getElementById('msg');
+
+const input = JSON.parse(localStorage.getItem('inputs'));
+
+if (input) {
+  nameField.value = input.name;
+  emailField.value = input.email;
+  messageField.value = input.message;
+}
+
+form.addEventListener('submit', (event) => {
+  event.preventDefault();
```

Comment:
- [ ] Please use the `change` event instead of the `submit` event, You should store the data in local storage every time there's a change on the form.

Reply:
Thank you for the value able feedback, I have implemented the given suggestion.
![Implement](https://user-images.githubusercontent.com/87594100/197224232-a29132f1-988e-4ffe-a3a7-18b1e3be6515.png)